File: src/main/java/com/paragon/client/systems/command/impl/SyntaxCommand.java
package com.paragon.client.systems.command.impl;

import com.paragon.Paragon;
import com.paragon.client.managers.CommandManager;
import com.paragon.client.systems.command.Command;
import net.minecraft.util.text.TextFormatting;

public class SyntaxCommand extends Command {

    public SyntaxCommand() {
        super("Syntax", "syntax [command]");
    }

    @Override
    public void whenCalled(String[] args, boolean fromConsole) {
        if (args.length == 1) {
            for (Command command : Paragon.INSTANCE.getCommandManager().getCommands()) {
                if (command.getName().equalsIgnoreCase(args[0])) {
                    CommandManager.sendClientMessage(command.getSyntax(), fromConsole);
                    break;
                }
            }
        } else {
            CommandManager.sendClientMessage(TextFormatting.RED + "Invalid syntax!", fromConsole);
        }
    }

}

File: src/main/java/com/paragon/client/systems/command/impl/SyntaxCommand.kt
package com.paragon.client.systems.command.impl

import com.paragon.Paragon
import com.paragon.client.systems.command.Command
import net.minecraft.util.text.TextFormatting

/**
 * @author Surge
 */
object SyntaxCommand : Command("Syntax", "syntax [command]") {

    override fun whenCalled(args: Array<String>, fromConsole: Boolean) {
        if (args.size == 1) {
            for (command in Paragon.INSTANCE.commandManager.commands) {
                if (command.name.equals(args[0], ignoreCase = true)) {
                    Paragon.INSTANCE.commandManager.sendClientMessage(command.syntax, fromConsole)
                    break
                }
            }
        } else {
            Paragon.INSTANCE.commandManager.sendClientMessage(TextFormatting.RED.toString() + "Invalid syntax!", fromConsole)
        }
    }

}